commands/scanandfixrepos.go

```diff
@@ -68,3 +85,57 @@ func (saf *ScanAndFixRepositories) downloadAndRunScanAndFix(repository *utils.Fr
 	cfp := CreateFixPullRequestsCmd{dryRun: saf.dryRun, dryRunRepoPath: filepath.Join(saf.dryRunRepoPath, repository.RepoName)}
 	return cfp.scanAndFixRepository(repository, branch, client)
 }
+
+func (saf ScanAndFixRepositories) setCommitBuildStatus(client vcsclient.VcsClient, repoConfig *utils.FrogbotRepoConfig, state vcsclient.CommitStatus, commitHash string, description string) error {
+	background := context.Background()
+	err := client.SetCommitStatus(background, state, repoConfig.RepoOwner, repoConfig.RepoName, commitHash, utils.FrogbotCreatorName, description, utils.CommitStatusDetailsUrl)
+	if err != nil {
+		log.Error("Failed to mark last commit as checked")
```

Comment:
Since we avoid adding error log messages when an error is returned, we should remove the above log message, but add it to the error that is returned below as follows:
```
return fmt.Errorf("failed to mark last commit as scanned due to: %s", err.Error())
```